app/lib/features/events/pages/event_page.dart

```diff
@@ -167,84 +193,194 @@ class CalendarEventPage extends ConsumerWidget {
-                                side: BorderSide(
-                                  color: Theme.of(context).colorScheme.success,
-                                  width: 1.5,
+                                child: Row(
+                                  mainAxisSize: MainAxisSize.min,
+                                  crossAxisAlignment: CrossAxisAlignment.start,
+                                  children: [
+                                    Text(
+                                      'Time: ',
+                                      style: Theme.of(context)
+                                          .textTheme
+                                          .bodyMedium,
+                                    ),
+                                    const SizedBox(width: 50),
+                                    Flexible(
+                                      flex: 2,
+                                      child: Text(
+                                        time,
+                                        style: Theme.of(context)
+                                            .textTheme
+                                            .bodyMedium,
+                                      ),
+                                    ),
+                                  ],
                                 ),
                               ),
-                            ),
-                            PopupMenuButton(
-                              tooltip: 'RSVP',
-                              offset: const Offset(80, 35),
-                              child: Container(
+                              Consumer(
+                                builder: (context, ref, child) {
+                                  // final profile = ref.watch(
+                                  //   spaceMemberProfileProvider(spaceMember),
+                                  // );
+                                  return Container(
+                                    alignment: Alignment.topLeft,
+                                    padding: const EdgeInsets.symmetric(
+                                      horizontal: 15,
+                                      vertical: 5,
+                                    ),
+                                    child: Row(
+                                      mainAxisSize: MainAxisSize.min,
+                                      crossAxisAlignment:
+                                          CrossAxisAlignment.start,
+                                      children: <Widget>[
+                                        Text(
+                                          'Host: ',
+                                          style: Theme.of(context)
+                                              .textTheme
+                                              .bodyMedium,
+                                        ),
+                                        const SizedBox(width: 50),
+                                        // profile.when(
```

Comment:
Is this meant to be removed or activated?

Reply:
It needs to be removed for now, as we need a feasible provider for giving member profile data from spaces.